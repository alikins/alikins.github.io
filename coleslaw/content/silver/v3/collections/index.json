{
    "meta": {
        "count": 1
    },
    "links": {
        "first": "example/content/silver/v3/collections/index.json",
        "last": "example/content/silver/v3/collections/index.json",
        "next": null,
        "prev": null
    },
    "data": [
        {
            "name": "netapp",
            "href": "example/content/silver/v3/collections/netapp/"
        }
    ]
}