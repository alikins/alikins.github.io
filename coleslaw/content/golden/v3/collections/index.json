{
    "meta": {
        "count": 3
    },
    "links": {
        "first": "example/content/golden/v3/collections/index.json",
        "last": "example/content/golden/v3/collections/index.json",
        "next": null,
        "prev": null
    },
    "data": [
        {
            "name": "alikins",
            "href": "example/content/golden/v3/collections/alikins/"
        },
        {
            "name": "ibm",
            "href": "example/content/golden/v3/collections/ibm/"
        },
        {
            "name": "netapp",
            "href": "example/content/golden/v3/collections/netapp/"
        }
    ]
}